pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md

```diff
@@ -1,3 +1,9 @@
+# 1.1.0
+2021-09-16 (Date of Last Commit)
+
+* Removed the Smart-seq2 Single Nucleus workflow (SmartSeq2SingleNucleus.wdl) and changed the workflow tasks to run multiple samples in the same VM
```

Comment:
maybe comment that this is expected to make the pipeline faster and cheaper

Reply:
done